### PR TITLE
Add Crimson Project Page

### PR DESCRIPTION
**Pull request by Matan-B**

### CODE REVIEW

File: src/en/news/crimson/index.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+regressions, but we need to update the test harness and test cases
+whenever it’s necessary. Also we need to
+
+- Review the performance to see if there is any regression not caught by the CI
+  tests.
+- Compare the performance of classic OSD and crimson-osd on monthly basis.
+
+* ### PG Scaling - Splits/Merges
+
+* ### Seastore
```
Comment (Matan-B) [inline src/en/news/crimson/index.md:67]: @cyx1231st, feel free to amend!

File: src/en/news/crimson/index.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+We are using a C++ framework named Seastar to implement Crimson, but we are not porting classic OSD to Seastar, as the design philosophy and various considerations could be very different between them. So, again, please bear in mind, to port classic OSD to Seastar and adapt it to bluestore is never our final goal.
+
+The new Crimson OSD will be compatible with classic OSD, so existing users will be able to upgrade from classic OSD to Crimson OSD. Put in other words, crimson-osd will:
+
+- Support librados protocol
+  - Be able to talk to existing librados clients
+  - Be able to join an existing Ceph cluster as an OSD
```
Comment (cyx1231st) [inline src/en/news/crimson/index.md:26]: It sounds like Crimson OSD will be able to co-exist and talk with classic OSDs in the same cluster?

I'm not aware of any design/discussions around the mixed mode.

File: src/en/news/crimson/index.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+- Log based recovery and Backfill
+- RBD workloads on Replicated pools
+- Bluestore, Seastore and cyanstore (memory-based) object store backends
+- Deployment via Cephadm
+
+Work in progeress:
+
+- Initial Scrub
+- EC pools
+- RGW
```
Comment (cyx1231st) [inline src/en/news/crimson/index.md:46]: Can we add:
- Multi-shard messenger/OSD/objectstores

I think this is still in progress.

File: src/en/news/crimson/index.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+- [Cephalocon 2023 - Crimson Project Update](https://www.youtube.com/watch?v=LaP4YX1lQ3I)
+- [From Classical to the Future](https://www.youtube.com/watch?v=8N_1WAEPw0o)
+- [Ceph Virtual 2022 - What's new with Crimson and Seastore?](https://www.youtube.com/watch?v=vc5w2mn93cY)
+- [Ceph Month 2021: Crimson Update](https://www.youtube.com/watch?v=vzJPOA7aJMk)
+- [Rebuilding a Faster OSD with Future](https://www.youtube.com/watch?v=3NmCPTJI1JE)
+- [Vault '20 - Crimson: A New Ceph OSD for the Age of Persistent Memory and Fast NVMe Storage](https://www.youtube.com/watch?v=FuFmMB9rbRA)
+- [A Glimpse of the New Ceph Messenger Built on Seastar](https://www.youtube.com/watch?v=X4Pz-dqrwi8)
+- [Error handling in a futurized environment](https://www.youtube.com/watch?v=HXfFX0wGKL4)
+- [Rebuilding the Ceph Distributed Storage Solution with Seastar](https://www.youtube.com/watch?v=A_25U4j7quA)
```
Comment (cyx1231st) [inline src/en/news/crimson/index.md:128]: A talk might be useful:
```
[Understanding SeaStore through profiling](https://www.youtube.com/watch?v=SUJjZ9bjXJc)
```

Which is an inital attempt to understand and optimize seastore (mostly from write amplification perspective).

**Review by cyx1231st — APPROVED** [review]

File: src/en/news/crimson/index.md (hunk 4 above)
Comment (Matan-B) [reply src/en/news/crimson/index.md:128]: Thanks!

File: src/en/news/crimson/index.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+- It will be able to bypass the kernel when talking to networking/storage devices which support polling.
+- It will be more computationally efficient, so the load is more balanced in a multi-core system, and each core will be able to serve the same amount of IO with lower CPU usage.
+- It will achieve these goals by a design from scratch. It also utilizes modern techniques, like SPDK, DPDK and the Seastar framework, so it can 1) bypass the kernel, 2) avoid memcpy between, 3) avoid lock contention.
+
+Crismon will be a drop-in-replacement of the classic OSD. Our long-term goal is to redesign the object storage backend from scratch, This new backend is dubbed “Seastore”. The disk layout we will use for Seastore won’t necessarily be optimal for HDDs. But Seastore is still in its early stage, and is still missing a detailed design. As an intermediate solution, bluestore is adapted to crimson-osd for testing purposes. But if the new object storage backend does not work well with HDD, we will continue using bluestore in crimson just for supporting HDD.
```
Comment (zdover23) [inline src/en/news/crimson/index.md:18]: s/Crismon/Crimson/

Comment (zdover23) [inline src/en/news/crimson/index.md:18]: s/from scratch,/from scratch./

Comment (zdover23) [inline src/en/news/crimson/index.md:18]: I'll just make an editorial pass over this after this PR is merged. Ignore these suggestions. I'll take care of them in the future.

Comment (Matan-B) [reply src/en/news/crimson/index.md:18]: Merged, thanks!

**Pull request merged**